src/test/java/com/wikia/webdriver/testcases/visualeditor/VEGalleryTests.java

```diff
@@ -40,16 +40,12 @@ public void VEGalleryTests_001_AddGallery() {
     ve.verifyVEToolBarPresent();
     ve.verifyEditorSurfacePresent();
     ve.typeTextArea(PageContent.ARTICLE_TEXT);
-    VisualEditorInsertGalleryDialog galleryDialog =
-        (VisualEditorInsertGalleryDialog) ve
-            .openDialogFromMenu(VisualEditorDataProvider.InsertDialog.GALLERY);
+    VisualEditorInsertGalleryDialog galleryDialog = ve.clickGalleryButton();
     galleryDialog = galleryDialog.searchMedia("he");
     ve = galleryDialog.addExistingMedia(numOfMedias);
     ve.verifyGalleries(numOfGalleries);
     ve.verifyMediasInGallery(numOfMedias);
-    VisualEditorSaveChangesDialog saveChangesDialog = ve.clickPublishButton();
-    article = saveChangesDialog.savePage();
-    article.verifyVEPublishComplete();
+    ArticlePageObject article = ve.publish();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "article" which hides the field declared at line 20. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AHiddenFieldCheck)